src/common/responses/AuthorizationError.yml

```
@@ -8,3 +8,7 @@
     application/json:
       schema:
         $ref: "../schemas/Error.yml"
+      examples:
+        tokenNotAuthorized:
+          summary: Token is not authorized to access the organization
```

I assume this error gets returned when trying to access other resources as well (write to a bucket, add a task, etc.)

```suggestion
          summary: Token is not authorized to access the organization or resource
```